README.md

```diff
@@ -65,8 +65,11 @@ Requests related to the funtionality or usability of the UI can be submitted as
     ```sh
     yarn test
     ```
-6. Code style:
-    * Not in IntelliJ: use an eslint plugin.
+6. Code style and linting:
+    * On command line within the repo's root dir:
+      `yarn run eslint src --fix`
+      * You can omit `--fix` if you want to only view linting errors.
+      * You can limit linting to specific file(s) by replacing `src` directory with file path(s) (e.g. `yarn run eslint src/data/machines.js --fix`)
```

Comment:
```suggestion
    * On command line within the repo's root dir: `yarn eslint <path> [--fix]`
      * To lint the main app files, use `src` as the path.
      * The `--fix` argument will tell eslint to fix as many violations as possible. It will report any that it can't fix itself.
    * In an IDE other than IntelliJ (VS Code, etc): install the eslint plugin from your package manager, and there should be a command to fix issues at any time.
```